client/src/ui/atoms/signin-link/index.tsx

```diff
@@ -14,10 +14,21 @@ export default function SignInLink({
   cta?: string;
 }) {
   const locale = useLocale();
+  const isServer = useIsServer();
   const gleanClick = useGleanClick();
-  const { pathname, search } = useLocation();
+  let { pathname, search } = useLocation();
   const sp = new URLSearchParams();
 
+  if (!isServer) {
+    // For some reason we need this :/
```

Comment:
I think this is because you set window.location directly, rather than using the react router functions to set the location - this is a bit smelly to me

Reply:
I don't think this is necessary any more, is it, since we don't modify search params anymore?